yabt/yabt.py

```diff
@@ -31,7 +31,8 @@
 from .cli import init_and_get_conf
 from .config import Config, BUILD_PROJ_FILE
 from .extend import Plugin
-from .graph import populate_targets_graph, write_dot
+from .graph import populate_targets_graph
+from .dot import write_dot
```

Comment:
sort imports

Reply:
what itamar said here + you need
- to change the doc below
- if I understood what's going on, there is no flag letting you see the buildenv (which you sometimes want), so after adding it, you need to add it in the help